### ISSUE TITLE
Suggestion: Compress video functionality

### ISSUE THREAD

**Issue by rangav** [pentarex/react-native-gallery-manager#3]
Hi 

I have a feature suggestion for videos.

Provide functionality to convert video to 720p or 1080p and also compress video quality option and save local to app.

Useful for sharing and etc..

**Comment by pentarex** [comment]
OK, I will see how exactly to do it, and try it.
I saw that you forked it, if you want to do it, please make pull request

**Comment by rangav** [comment]
I am still doing research on this,  if i found a solution then happy to implement and submit a PR. will update you..

**Comment by pentarex** [comment]
Ive started implementing some of this functionality.
the method will be called convertVideo
the params will be 

id: Local id of the video asset,
compressType: [check preset names](https://developer.apple.com/documentation/avfoundation/avassetexportsession/1389367-initwithasset?language=objc)
convertTo: [some of those only](https://developer.apple.com/documentation/avfoundation/avfiletype?language=objc), I think AVFileTypeMPEG4, AVFileTypeAppleM4V, AVFileTypeQuickTimeMovie
tempName: the temporary name of the asset.

where compressType, convertTo, tempName are going be optional.
So the client can choose to convert or to compress or both

It will return probably the same stuff as now (if makes sense)

```objective-c
NSArray * _Nonnull localIds = [NSArray arrayWithObjects: localIdentifier];
  
  PHFetchOptions *fetchOptions = [[PHFetchOptions alloc] init];
  
  PHAssetCollection * _Nullable video = [PHAssetCollection fetchAssetCollectionsWithLocalIdentifiers:localIds options:fetchOptions].firstObject;
  
  [AVAssetExportSession exportSessionWithAsset:video presetName:AVAssetExportPresetPassthrough];
```

tell me what you think

**Comment by rangav** [comment]
Looks good buddy to me, Please go ahead and publish it, I will test it and let you know any suggestions.

**Comment by pentarex** [comment]
Probably end of the week.

**Comment by pentarex** [comment]
implemented in 1.0.5
